File: SDKs/twitter.com/labs/1.14/python/src/sdk/models/shared/deleterulesresponse.py
from dataclasses import dataclass, field
from dataclasses_json import dataclass_json
from . import rulesresponsemetadata


@dataclass_json
@dataclass
class DeleteRulesResponse:
    meta: rulesresponsemetadata.RulesResponseMetadata = field(default=None, metadata={'dataclasses_json': { 'field_name': 'meta' }})
    

File: SDKs/twitter.com/labs/1.14/typescript/src/sdk/models/shared/deleterulesresponse.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";
import { RulesResponseMetadata } from "./rulesresponsemetadata";



export class DeleteRulesResponse extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=meta" })
  meta: RulesResponseMetadata;
}
